Maddori.Apple/Maddori.Apple/Screen/Setup/SetNickname/SetNicknameViewController.swift

```diff
@@ -328,17 +355,24 @@ final class SetNicknameViewController: BaseViewController {
     
     // MARK: - api
     
-    private func dispatchCreateTeam(type: SetupEndPoint<CreateTeamDTO>) {
-        AF.request(type.address,
-                   method: type.method,
-                   parameters: type.body,
-                   encoder: JSONParameterEncoder.default,
-                   headers: type.headers
-        ).responseDecodable(of: BaseModel<CreateTeamResponse>.self) { json in
+    private func dispatchCreateTeam(type: SetupEndPoint<EncodeDTO>, teamName: String, nickname: String, role: String?) {
+        AF.upload(multipartFormData: { multipartFormData in
+            let teamInfo: Dictionary = ["team_name": teamName, "nickname": nickname, "role": role]
```

Comment:
p3;

조금 더 읽기 편하게 줄바꿈 부탁드립니다!

```suggestion
            let teamInfo: Dictionary = ["team_name": teamName,
                                        "nickname": nickname,
                                        "role": role]
```

Reply:
[0532980](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/315/commits/0532980581c9663d2ecea39a0a3ba803632e2144)
수정완료 ⭐️!!!!